differential-expression/README.md

```diff
@@ -1,25 +1,100 @@
+We recommend using the [`limma` user guide](https://www.bioconductor.org/packages/devel/bioc/vignettes/limma/inst/doc/usersguide.pdf)
+to help you set up your model in a way that takes into account your experimental
+set up and hypotheses.
 
 ***
-### Differential Expression Analyses in GenePattern
+## Differential expression analysis with GenePattern
+
+[GenePattern notebooks](http://genepattern-notebook.org/example-notebooks/)
+contain [ready-made analyses](http://genepattern-notebook.org/example-notebooks/).
+For users who are more comfortable with Python, or are not comfortable with
+using R Notebooks, the GenePattern notebooks are written in Python but can be run using a graphics user interface (GUI).
+To use GenePattern, you have to create an account.
+Here is [their guide](http://software.broadinstitute.org/cancer/software/genepattern/quick-start) we recommend you follow to get started.
+For use with GenePattern notebooks, data from refine.bio needs to be converted to GenePattern
+[formats](http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide).
+If you would like to do differential expression but would prefer using GenePattern, follow the instructions below.
+
+### Preparing your refine.bio data for GenePattern
+
+In order to complete your differential expression analysis using [GenePattern](https://cloud.genepattern.org/gp/pages/login.jsf), you will need to have:  
+ 1) a GCT file    
+ 2) a CLS file  
+
+You can follow the steps below to create these files from your refine.bio data.
+
+#### Create a GCT file
+
+Convert a gene expression tab separated values (TSV) file provided into a 'gene cluster text' (GCT) file for use in GenePattern notebooks.
+In order to create a GCT formatted file from a TSV refine.bio data file, download and reference the
+[`create_gct_file.R` script](https://github.com/AlexsLemonade/refinebio-examples/blob/master/scripts/create_gct_file.R), followed by `--file` argument with the name of the refine.bio dataset TSV file in your current directory that you would like to convert.
+Note: This script requires `optparse` library. If `optparse` library is not installed, this script will install it for you.
+Be sure to either have the script and input file in your current working directory, or put type out the full directory path for the script and/or input file. eg. `/users/Bob/Desktop/scripts/create_gct_file.R`
+  
+If you need more guidance on how to navigate directories, we recommend [this tutorial](https://swcarpentry.github.io/shell-novice/02-filedir/index.html).
+
+Options:     
+`--file` :name of the file in your current directory that you would like to convert.    
+`--rewrite` :file of the same name as the output will be rewritten (*optional*)       
+`--output` :name for the output file, the ".gct" suffix will be added if you do not add it yourself  (*optional*)
+
+##### Examples of usage in command line:  
```

Comment:
I think this needs more examples that demonstrate what happens if you don't use `--output` and what `--rewrite` is about. The request was for multiple examples.